src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,
+        target: ScriptTarget.ES3,
+        noImplicitAny: true,
+        outDir: "built",
+        rootDir: ".",
+        sourceMap: false,
+    }
+
+    export function buildConfigFile(writer: EmitTextWriter, compilerOptions: CompilerOptions, fileNames?: string[], excludes?: string[]) {

```

Comment:
`commandLineParser.ts` host the code for parsing logic, so I don't think it fits in their either. What do you think about creating a new file `configEmitter.ts`? That's probably more inline with `declarationEmitter.ts`.